panic: runtime error: invalid memory address or nil pointer dereference

I have encountered the above panic situation recently.  This happens after a long while that podsync was not turned on.   After podsync is turned back on, it tries to download >50 youtube video and to convert them to high-res audio.

This panic situation happens randomly after a certain number of youtube video has been downloaded and converted.   I put down below part of the log:

`time="2021-11-18T16:44:39+08:00" level=info msg="\n _______ _______ ______ _______ _ _______ \n( ____ )( ___ )( __ \\ ( ____ \\|\\ /|( ( /|( ____ \\\n| ( )|| ( ) || ( \\ )| ( \\/( \\ / )| \\ ( || ( \\/\n| (____)|| | | || | ) || (_____ \\ (_) / | \\ | || | \n| _____)| | | || | | |(_____ ) \\ / | (\\ \\) || | \n| ( | | | || | ) | ) | ) ( | | \\ || | \n| ) | (___) || (__/ )/\\____) | | | | ) \\ || (____/\\\n|/ (_______)(______/ \\_______) \\_/ |/ )_)(_______/\n"
time="2021-11-18T16:44:39+08:00" level=info msg="running podsync" commit=d39e464585a4364b1f74808975f7945182f209b7 date="2020-10-17T23:50:21Z" version=2.4.0
time="2021-11-18T16:44:41+08:00" level=info msg="using youtube-dl 2021.06.06\n"
time="2021-11-18T16:44:41+08:00" level=info msg="found ffmpeg: ffmpeg version 4.1.6 Copyright (c) 2000-2020 the FFmpeg developers\nbuilt with gcc 8.3.0 (Alpine 8.3.0)\nconfiguration: --prefix=/usr --enable-avresample --enable-avfilter --enable-gnutls --enable-gpl --enable-libass --enable-libmp3lame --enable-libvorbis --enable-libvpx --enable-libxvid --enable-libx264 --enable-libx265 --enable-libtheora --enable-libv4l2 --enable-postproc --enable-pic --enable-pthreads --enable-shared --enable-libxcb --disable-stripping --disable-static --disable-librtmp --enable-vaapi --enable-vdpau --enable-libopus --disable-debug\nlibavutil 56. 22.100 / 56. 22.100\nlibavcodec 58. 35.100 / 58. 35.100\nlibavformat 58. 20.100 / 58. 20.100\nlibavdevice 58. 5.100 / 58. 5.100\nlibavfilter 7. 40.101 / 7. 40.101\nlibavresample 4. 0. 0 / 4. 0. 0\nlibswscale 5. 3.100 / 5. 3.100\nlibswresample 3. 3.100 / 3. 3.100\nlibpostproc 55. 3.100 / 55. 3.100\n"
time="2021-11-18T16:44:41+08:00" level=info msg="updating youtube-dl"
time="2021-11-18T16:44:42+08:00" level=info msg="[debug] System config: []\n[debug] User config: []\n[debug] Custom config: []\n[debug] Command-line args: [u'--update', u'--verbose']\n[debug] Encodings: locale UTF-8, fs UTF-8, out None, pref UTF-8\n[debug] youtube-dl version 2021.06.06\n[debug] Python version 2.7.18 (CPython) - Linux-5.10.28-Unraid-x86_64-with\n[debug] exe versions: ffmpeg 4.1.6, ffprobe 4.1.6\n[debug] Proxy map: {}\nyoutube-dl is up-to-date (2021.06.06)\n"
time="2021-11-18T16:44:42+08:00" level=info msg="opening database \"db\""
time="2021-11-18T16:44:43+08:00" level=info msg="All 1 tables opened in 14ms\n"
time="2021-11-18T16:44:43+08:00" level=info msg="Replaying file id: 0 at offset: 110663802\n"
time="2021-11-18T16:44:43+08:00" level=info msg="Replay took: 166.487ms\n"
time="2021-11-18T16:44:43+08:00" level=info msg="running listener at :8080"
time="2021-11-18T16:44:43+08:00" level=info msg="-> updating https://www.youtube.com/channel...removed..." feed_id=martinoei format=audio quality=high
time="2021-11-18T16:44:46+08:00" level=info msg="downloading episodes" page_size=50
time="2021-11-18T16:44:46+08:00" level=info msg="download count: 49"
time="2021-11-18T16:44:46+08:00" level=info msg="! downloading episode https://youtube.com/watch?v=...removed..." episode_id=...removed... index=0
time="2021-11-18T16:51:39+08:00" level=info msg="successfully downloaded file \"...removed...\"" episode_id=-...removed... index=0
time="2021-11-18T16:51:39+08:00" level=info msg="! downloading episode https://youtube.com/watch?v=...removed..." episode_id=...removed... index=1
time="2021-11-18T16:56:44+08:00" level=info msg="successfully downloaded file \"...removed...\"" episode_id=...removed... index=1
time="2021-11-18T16:56:44+08:00" level=info msg="! downloading episode https://youtube.com/watch?v=...removed..." episode_id=...removed...U index=2
time="2021-11-18T17:02:32+08:00" level=info msg="successfully downloaded file \"...removed...\"" episode_id=...removed... index=2
time="2021-11-18T17:02:32+08:00" level=info msg="! downloading episode https://youtube.com/watch?v=...removed..." episode_id=...removed... index=3
time="2021-11-18T17:09:43+08:00" level=info msg="successfully downloaded file \"...removed...\"" episode_id=...removed... index=3
time="2021-11-18T17:09:43+08:00" level=info msg="! downloading episode https://youtube.com/watch?v=...removed..." episode_id=...removed... index=4
time="2021-11-18T17:18:04+08:00" level=info msg="successfully downloaded file \"...removed...\"" episode_id=...removed... index=4
time="2021-11-18T17:18:04+08:00" level=info msg="! downloading episode https://youtube.com/watch?v=...removed... episode_id=...removed... index=5
time="2021-11-18T17:26:38+08:00" level=info msg="successfully downloaded file \"...removed...\"" episode_id=...removed... index=5
time="2021-11-18T17:26:38+08:00" level=info msg="! downloading episode https://youtube.com/watch?v=...removed..." episode_id=...removed... index=6
time="2021-11-18T17:33:31+08:00" level=info msg="successfully downloaded file \"...removed...\"" episode_id=...removed... index=6
time="2021-11-18T17:33:32+08:00" level=info msg="! downloading episode https://youtube.com/watch?v=...removed..." episode_id=...removed... index=7
time="2021-11-18T17:39:52+08:00" level=info msg="successfully downloaded file \"...removed...\"" episode_id=...removed... index=7
time="2021-11-18T17:39:53+08:00" level=info msg="! downloading episode https://youtube.com/watch?v=...removed..." episode_id=...removed... index=8
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x20 pc=0x737f09]

goroutine 828 [running]:
github.com/robfig/cron/v3.SkipIfStillRunning.func1.1()
/home/runner/go/pkg/mod/github.com/robfig/cron/v3@v3.0.1/chain.go:88 +0xa9
github.com/robfig/cron/v3.FuncJob.Run(0xc000182330)
/home/runner/go/pkg/mod/github.com/robfig/cron/v3@v3.0.1/cron.go:136 +0x25
github.com/robfig/cron/v3.(*Cron).startJob.func1(0xc000278000, 0xb51cc0, 0xc000182330)
/home/runner/go/pkg/mod/github.com/robfig/cron/v3@v3.0.1/cron.go:312 +0x65
created by github.com/robfig/cron/v3.(*Cron).startJob
/home/runner/go/pkg/mod/github.com/robfig/cron/v3@v3.0.1/cron.go:310 +0x73`

--- a/cmd/podsync/main.go
+++ b/cmd/podsync/main.go
@@ -138,7 +138,7 @@ func main() {
 	defer close(updates)
 
 	// Create Cron
-	c := cron.New(cron.WithChain(cron.SkipIfStillRunning(nil)))
+	c := cron.New(cron.WithChain(cron.SkipIfStillRunning(cron.DiscardLogger)))
 	m := make(map[string]cron.EntryID)
 
 	// Run updates listener

